c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -693,13 +715,19 @@ else
 endif
 
 $(NPM_CMD):
+ifeq ($(OPERATING_SYSTEM), WINDOWS)
```

Comment:
@sbrunner, really not sure about this...

We had defined that on Wednesday:
```
$(NPM_CMD):
ifeq ($(OPERATING_SYSTEM), LINUX)
	npm install npm
endif

.build/node_modules.timestamp: package.json $(NPM_CMD)
	mkdir -p $(dir $@)
ifeq ($(NGEO), TRUE)
ifeq ($(OPERATING_SYSTEM), LINUX)
	$(NPM_CMD) install
endif
endif
	touch $@
```

But if I do so, then npm does not get installed when doing a fresh install... Moreover, I really do not get what these two sections really do...